ci/record_rep_weights.py

```diff
@@ -0,0 +1,56 @@
+import requests
+import argparse
+import string
+from binascii import unhexlify
+from base64 import b32decode
+from binascii import hexlify, unhexlify
+
+
+parser = argparse.ArgumentParser(description='Generate bootstrap representative weight file.')
+parser.add_argument("output", type=str, help="output weight file")
+parser.add_argument("--rpc", help="node rpc host:port", default="http://[::1]:7076")
+parser.add_argument("--limit", help="percentage of the active supply represented", default=0.95)
+parser.add_argument("--cutoff", help="stop using bootstrap reps this many blocks before the current block height", default=500000)
+args = parser.parse_args()
+
+r = requests.post(args.rpc, data='{"action":"representatives"}')
+p = r.json()
+
+reps = [ ]
+tbl = string.maketrans('13456789abcdefghijkmnopqrstuwxyz', 'ABCDEFGHIJKLMNOPQRSTUVWXYZ234567')
+for acc in p["representatives"]:
+    reps.append({
+        'account': acc,
+        'weight': long(p["representatives"][acc])
+    })
+
+r = requests.post(args.rpc, data='{"action":"block_count"}')
+p = r.json()
+block_height = max(0, int(p["count"]) - args.cutoff)
+
+print("cutoff block height is %d" % block_height)
```

Comment:
If you're going to use py3k style print, then I would recommend a `from __future__ import print_function` at the top.  And if you do end up adding that, could you put a shebang with `/usr/bin/env python` too?

Reply:
This is just due to pure python ignorance, changed.